site/src/api/queries/workspaces.ts

```diff
@@ -61,15 +54,41 @@ export const createWorkspace = (queryClient: QueryClient) => {
   };
 };
 
+type AutoCreateWorkspaceOptions = {
+  organizationId: string;
+  templateName: string;
+  name: string;
```

Comment:
Could this be renamed to `workspaceName` to make it more clear how it's different from `templateName`?